csharp-advanced-topics/EventsDrivenArchitectureInCsharp/SharedLibrary/ThermostatTempChangeEvent.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Events;
+
+public class ThermostatTempChangeEvent
+{
+    public Guid CorrelationId { get; set; }
+    public decimal NewTemperature { get; set; }
```

Comment:
Instead of `NewTemperature`, I would call this property just `Temperature`. Since it is the event and it has the `Change` word in its name, it's implicitly clear that this will correspond to the "new" temperature.